message_ix/tests/test_equation_NEW_CAPACITY_CONTRAINT.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from message_ix.testing import make_westeros
+
+
+@pytest.mark.parametrize(
+    "model_horizon",
+    [
+        [[700, 710, 715, 720], [3.746712, 4.149163, 8.731826, 9.182337]],
```

Comment:
It will be useful to mention what these values are corresponding to each year, and perhaps how they are derived (e.g., calculation steps or showing the equation used for deriving them). I assume these are maximum `CAP_NEW` that can be installed for `wind_ppl` depending on certain input values for "initial-" and "growth_new_capacity_up".

Reply:
Thanks Behnam. The formula used to derive these is documented as part of MESSAGEix code so I dont see why we should repeat this here; it is a simple matter of using the "CAP_NEW" equation and the parameters of the technology to derive the values. I think this is relatively straight-forward. I have also stated in the documentation of the test itself, that we are checking the calculation of "CAP_NEW".